migrator/db.py

```diff
@@ -356,6 +356,7 @@ def temp_db_url(control_conn: Any) -> Iterator[str]:
     db_name = "".join(random.choices("qwertyuiopasdfghjklzxcvbnm", k=10))
     cur.execute("CREATE DATABASE " + db_name)
     try:
+        # TODO(JJ): use the URL passed in control_conn
```

Comment:
as Jesper pointed out I think this code should be using connection details that are passed in the control_conn and not the DATABASE_URL